workspace/utilities/lib/share-link.xsl

```diff
@@ -41,7 +41,7 @@
 		<xsl:variable name="computed-url">
 			<xsl:choose>
 				<xsl:when test="string-length($status) != 0">
-					<xsl:value-of select="concat('?subject=', $encoded-status, $separator,  '&amp;body=', $encoded-body)" />
+					<xsl:value-of select="concat('?subject=', $encoded-status, '&amp;body=', $encoded-body)" />
```

Comment:
It should be removed from the parameters as well

Reply:
Done